wiki/Game_modifier/No_Fail/id.md

```diff
@@ -0,0 +1,36 @@
+---
+stub: true
+tags:
+  - no fail
+  - NF
+  - mod
+  - game modifier
+---
+
+# No Fail (mod)
+
+![Ikon mod NF](/wiki/shared/mods/NF.png "Ikon mod No-Fail (NF)")
+
+*Untuk melihat daftar keseluruhan dari semua mod, kunjungi: [Game modifier](/wiki/Game_modifier)*
+
+## Tentang
+
+- Singkatan: NF
+- Tipe: Pengurang Kesulitan
+- Pengali Skor: 0.50x
+- Tombol shortcut default: `W`
+- Caption: `Anda tak mungkin gagal. Tak peduli apapun yang terjadi..`
+- Mode-mode permainan yang cocok: ![][osu!] ![][osu!taiko] ![][osu!catch] ![][osu!mania]
+
+## Deskripsi
+
+*Perhatian: Penggunaan mod No Fail akan mengurangi jumlah [Performance Point](/wiki/Performance_Points) yang dapat diraih sebesar 10%.*
```

Comment:
o i believe this is no longer the case since http://osu.ppy.sh/home/news/2021-01-14-performance-points-updates but that's another issue that is not in the scope of this pr i guess...

![image](https://user-images.githubusercontent.com/36564236/123310797-aeb30580-d550-11eb-8b0e-aca340703948.png)

Reply:
yeah, still thinking about making a quick update to the EN article to something like "Notice: Usage of the No Fail mod reduces the amount of possible Performance Points after specific amount of misses, see this article for more regarding this: (link goes here)"